### DISCUSSION TITLE
Graduate Thesis - English version

### DISCUSSION THREAD

**Discussion by Alkrick** [BITNP/BIThesis#269] [Q&A] [answered]
Is there a dedicated english template for the graduate thesis? I have been using the Chinese one but I can't figure out how to remove the "第n章" at the begining of each chapter from the main text and from the table of contents. Any help would be appreciated!

**Comment by fky2015** [comment, marked as answer]
I think there is a lot to be discussed about the English mode in Graduate Thesis.

## Is there a dedicated English template?

Yes. From version `v3.4.3` ~~(**not released yet**)~~, you can switch on the option `english=true` in `\documentclass`.
Then you will see `Chapter n` instead of `第n章`.

## Is the English mode template ready to be used as the dissertation?

I cannot provide a confident answer since we do not have any official template to proofread further.
I would be appreciated it if you could provide some help.

But in a larger sense, all functionalities provided in Chinese mode are also available in English mode.

## Why is this title/content still in Chinese?

Feel free to file an issue on Github and give us feedback. Rome wasn't built in a day!
